devServer.js

```diff
@@ -2,7 +2,7 @@ const path = require("path");
 const express = require("express");
 const webpack = require("webpack");
 
-const server = process.env.RJSF_DEV_SERVER || "localhost:8080";
```

Comment:
Why not just override the `RJSF_DEV_SERVER` value in the .env file?

Reply:
Ups, forgot to remove that change.

Reply:
Fixed.